animations/README.md

```diff
@@ -39,6 +39,7 @@ it using a `PageRouteBuilder`.
 spring simulation.
 - **CardSwipeDemo** A swipeable card that demonstrates how to use gesture
 detection to drive an animation.
+- **AnimatedList** Demonstrates how to use `AnimatedList`
```

Comment:
You may need to rebase your changes.

Reply:
@RedBrogdon I intentionally made this change cause I noticed `AnimatedList` demo has been moved into the misc directory - https://github.com/flutter/samples/blob/master/animations/lib/src/misc/animated_list.dart but not updated in the Readme. 
Should I revert this anyway though? Let me know.